src/localization/modalStrings.js

```diff
@@ -57,6 +57,8 @@ export const modalStrings = new LocalizedStrings({
     select_master_lists: 'Select master list',
     customer_no_masterlist_available: 'This customer has no master lists!',
     supplier_no_masterlist_available: 'Your store has no master lists!',
+    emergency_orders_can_only_have: 'Emergency orders can only have',
+    items_remove_some: 'items! Remove some before you can finalise',
```

Comment:
@alainsussol Can I get french translations pretty please?

- Emergency orders can only have
- items! Remove some before you can finalise

Reply:
> Emergency orders can only have

Commandes urgentes peuvent n'avoir que


> items! Remove some before you can finalise

articles ! Enlevez-en quelques-uns avant de finaliser.

Reply:
Thanks so much!!